### PR TITLE
feat: astro-font - optimize fonts & prevent CLS

### PR DESCRIPTION
**Pull request by rishi-raj-jain**
Read more on `astro-font`: https://www.launchfa.st/features/astro-font

### CODE REVIEW

**Comment by thatgurkangurk** [comment]
i'll check it out later but looks good

File: src/components/meta/Meta.astro [hunk 1]
```diff
@@ -14,18 +15,26 @@ const { title } = Astro.props;
...
+          style: "normal",
+          weight: "700",
+          path: "./public/fonts/GeistMono.woff2",
+        },
+        {
+          style: "normal",
+          weight: "700",
+          path: "./public/fonts/GeistMono.ttf",
+        },
+      ],
```
Comment (thatgurkangurk) [inline src/components/meta/Meta.astro:37]: the weight should be 70, not 700

**Review by thatgurkangurk — CHANGES_REQUESTED** [review]
the weight is 70, not 700

**Comment by thatgurkangurk** [comment]
when this is fixed feel free to re-request a review :)

**Comment by rishi-raj-jain** [comment]
@thatgurkangurk 

70 is not a valid font weight: https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight

**Comment by rishi-raj-jain** [comment]
I wonder where you found that? How about using 100?

**Comment by thatgurkangurk** [comment]
huh, alright try 100 and send a screenshot (_since cloudflare apparantly doesnt deploy from pull requests_)

**Comment by thatgurkangurk** [comment]
> I wonder where you found that? How about using 100?

i'm pretty sure that i just randomly chose something that looked good 🤣

**Comment by rishi-raj-jain** [comment]
Screenshot 2024-01-12 at 10 39 09 PM
Screenshot 2024-01-12 at 10 39 16 PM

**Comment by thatgurkangurk** [comment]
looks good

**Comment by thatgurkangurk** [comment]
thanks for your contribution

**Pull request merged**